src/utils/common.js

```diff
@@ -0,0 +1,126 @@
+import dayjs from 'dayjs';
+import { Keydown } from '../enums';
+import AbstractView from '../view/abstract';
+
+const getRandomInteger = (a = 0, b = 1) => {
+  const lower = Math.ceil(Math.min(a, b));
+  const upper = Math.floor(Math.max(a, b));
+
+  return Math.floor(lower + Math.random() * (upper - lower + 1));
+};
+
+const humanizeEventStartDate = (dateStr) => dayjs(dateStr).format('MMM DD');
+
+const sortDateUp = (a, b) => {
+  if (a.isAfter(b)) {
+    return 1;
+  }
+  return -1;
```

Comment:
```suggestion
  return a.isAfter(b) ? 1 : -1;
```
Так можно в нескольких местах сделать. Давай сразу по критериям будем делать)

Reply:
Исправлю

Reply:
Done